projects/igniteui-angular/src/lib/drop-down/drop-down-item.base.ts

```diff
@@ -196,6 +212,22 @@ export abstract class IgxDropDownItemBase implements DoCheck {
     @HostBinding('class.igx-drop-down__header')
     public isHeader: boolean;
 
+    /**
```

Comment:
drop down could update its items display density via hidden input. This way items will not know about drop down. Other possible solution is to use some service to update display density of the items.

Reply:
I believe this would be better left for another PR - in order for the drop-down item to not know about it's parent, the remaining selection logic and the `allowItemFocus` handling inside of the drop-down-item component will have to be abstracted in a service as well